### PR TITLE
Update build.gradle

### PR DESCRIPTION
**Pull request by ahndmal**
Spring Boot version upgrade in build.gradle

### CODE REVIEW

**Comment by ahndmal** [comment]
Hey! created the PR with Spring Boot version upgrade in build.gradle

**Comment by ahndmal** [comment]
Hi @jeschkies ! Should I close the pull request ?
Thanks

**Comment by jeschkies** [comment]
I'm sorry. I didn't get notified. Let me take a look.

File: .settings/org.eclipse.buildship.core.prefs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+arguments=
```
Comment (jeschkies) [inline .settings/org.eclipse.buildship.core.prefs]: I don't think we need these. Would you mind removing them?

File: .project [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jeschkies) [inline .project]: Same here. Could you remove it?

**Comment by ahndmal** [comment]
Hello @jeschkies ,
Applied the changes. Please check :)

**Review by jeschkies — APPROVED** [review]

**Pull request merged**